app/assets/javascripts/discourse/app/components/modal/ignore-duration-with-username.js

```diff
@@ -16,6 +16,14 @@ export default class IgnoreDurationModal extends Component {
   @tracked ignoredUntil = null;
   @tracked ignoredUsername = null;
 
+  enableSelection = true;
+
+  constructor() {
+    super(...arguments);
+    this.ignoredUsername = this.args.model.ignoredUsername;
+    this.enableSelection = this.args.model.enableSelection ?? true;
+  }
+
```

Comment:
It can be simplified to:

```suggestion
  enableSelection = this.args.model.enableSelection ?? true;
  ignoredUsername = this.args.model.ignoredUsername;

```

native class field initialization ftw! 😃

Reply:
Thanks, @CvX! This is exactly what I was looking for. 💌